.snyk

```diff
@@ -0,0 +1,6 @@
+exclude:
+  global:
+    - .github/**
+    - docs/**
+    - pkg/**/*_test.go
+    - tests/**
```

Comment:
i'm not sure that we should exclude this folder. These tests are public, meaning: anyone can import them, plug their own datastore, and run them.

Reply:
👍 Good call, I see your point. Thanks for the added context and link for background. I'll scrap that line.